openfe/analysis/plotting.py

```diff
@@ -278,15 +294,15 @@ def plot_2D_rmsd(data: list[list[float]],
     fig, axes = plt.subplots(nrows, 4)
 
     for i, (arr, ax) in enumerate(
-            zip(twod_rmsd_arrs, chain.from_iterable(axes))):
+            zip(twod_rmsd_arrs, axes.flatten())):   # type: ignore
```

Comment:
for some reason the return type for axes can be Axes | ndarray, but in our use case we always have ndarray (because we set ncols to 4).